R/navbar.R

```diff
@@ -28,9 +28,15 @@ data_navbar <- function(pkg = ".", depth = 0L) {
     )
   }
 
+  navbar_bg <- if (is.null(get_bootswatch_theme)) {
```

Comment:
this is to iterate faster if there is no bootswatch theme, to have the color decided by CSS. However the type is still decided by the HTML.